_themes/launcher.md

```diff
@@ -0,0 +1,18 @@
+---
+heading: Launcher
+thumbnail: themes/images/thumbnails/launcher.jpg
+categories:
+  - single-page
+description: For those of us who aren't ready yet.
```

Comment:
I'd like to change this so it's at least a little different than [Launch](http://picocms.org/themes/#entry-launch).

I'm thinking maybe "Launching in 3... 2... 1...".

These were originally meaningful descriptions, but at some point I just started writing some cheeky/fun text. :sweat_smile: